CMakeLists.txt

```diff
@@ -23,26 +35,19 @@ if(NOT CMAKE_BUILD_TYPE)
             "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel." FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
-# option for more verbose log messages.
-option(VERBOSE_LOGGING "Print Filename and line in every log line." OFF)
-if (VERBOSE_LOGGING)
-    add_definitions(-DAUTOPAS_VERBOSE_LOG)
-endif ()
-
 ENABLE_TESTING()
 
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules/)
 
-include(other-compileroptions)
-include(vectorization)
+include(clang-format)
 include(clang-sanitizers)
-include(version.cmake)
 include(clang-tidy)
-include(clang-format)
+include(CodeCoverageAutopas)
 include(doxygen)
+include(logging)
```

Comment:
eventually move logging to `modules needing targets`

Reply:
done e3c1c0f3e68e09a62b69b7b4856fa9010181f9ae